Source/ASNodeController+Beta.h

```diff
@@ -10,18 +10,21 @@
 #import <AsyncDisplayKit/ASDisplayNode.h>
 #import <AsyncDisplayKit/ASDisplayNode+Subclasses.h> // for ASInterfaceState protocol
 
+NS_ASSUME_NONNULL_BEGIN
+
 /* ASNodeController is currently beta and open to change in the future */
 @interface ASNodeController<__covariant DisplayNodeType : ASDisplayNode *>
-    : NSObject <ASInterfaceStateDelegate, NSLocking>
+    : NSObject <ASInterfaceStateDelegate>
```

Comment:
Should we actually remove NSLocking here? Or can -lock/-unlock just do the right thing?

Reply:
It's definitely arguable. I like less code, and more transparency, but I don't feel strongly about it.

Reply:
It's back in